mistletoe/token.py

```diff
@@ -1,5 +1,5 @@
 """
-Base token class
+token contains the base token class Token.
```

Comment:
I have quickly gone through the various files inside this "mistletoe" directory. For consistency, I would vote for changing this to:

    Base token class.

Or we could possibly write `Common code for token classes.`, but I would keep it short either way.

Reply:
OK, I've updated the module docstring.

Reply:
thanks